.github/workflows/release.yml

```diff
@@ -0,0 +1,75 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs: 
+      current_version:
+        description: 'current version number'
+        required: true
+        type: string
+      release_version:
+        description: 'release version number'
+        required: true
+        type: string
+
+jobs:
+  release:
+    env:
+      CURRENT_VERSION: ${{ github.event.inputs.current_version }}
+      RELEASE_VERSION: ${{ github.event.inputs.release_version }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      
+      - name: git config
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "SpringQL release script"
+
+      - name: edit changelog
+        run: |
+          # load helper script
+          source .github/workflows/scripts/keep-a-changelog.bash
+          # edit CHANGELOG.md
+          cp CHANGELOG.md CHANGELOG.md.org
+          cat CHANGELOG.md.org | bump_changelog "${CURRENT_VERSION}" "${RELEASE_VERSION}" > CHANGELOG.md
+          rm CHANGELOG.md.org
+      
+      - name: bump crate versions
+        run: |
+          cd springql
+          cargo bump "${RELEASE_VERSION}"
```

Comment:
Does `cargo-bump` update dependency versions?

```toml
# springql/Cargo.toml
# ...
[dependencies]
springql-core = "0.14.0"
```

```bash
cd springql-core
cargo bump 0.15.0
```

Then will the "= 0.14.0" be changed into "= 0.15.0"?

Reply:
`${RELEASE_VERSION}` contains `v` prefix. Is that OK?

Reply:
cargo-bump is here. https://crates.io/crates/cargo-bump

Reply:
Yeah, I read the document but I could not figure out these questions from it.

Reply:
`${RELEASE_VERSION}' and `${CURRENT_VERSION}` must not contains `v`.

`cargo-bump` updates crate package version only. dependencies not changed.
dependency update needs `cargo upgrade` https://crates.io/crates/cargo-edit#cargo-upgrade